### ISSUE TITLE
great little and usefull program thank you

### ISSUE THREAD

**Issue by m040601** [sgerwk/multiselect#1]
Hi! This is just to give you some feedback, suggestions and praise about this little interesting and usefull program. (Found it on your github profile, via your comments on felinks repo).

I compiled and runit without any problems. Loved it at first sight. Great thing that you provided a usefull man page with use cases. 

1. package for Arch

I see you have also packged your other program pdftoroff/hovacui (also great and usefull)  program for Arch, https://aur.archlinux.org/packages/pdftoroff

Could you also make an AUR for multiselect ?

2. Describe it better on the README on github

Make it more prominent how usefull this program can be.
Put the stuff you have on the man page on the README.
You have a great example for filing forms with a comma separated file:

cat data.txt | 
    while read A;
    do
        echo "$A" | tr ',' '\\n' | multiselect
    done

I'm a power command line user, but just by glancing at the README page, I never thought of this. So usefull. Maybe you can come up with other examples.

3. I'm curious, but I'm not  a programmer. 

It had never occurred to me that such usefull things could be made with a "simple" one file program in C. Using "just" these libraries
                                                                                
#include <stdlib.h>                                                             
#include <stdio.h>                                                              
#include <string.h>                                                             
#include <sys/time.h>                                                           
#include <X11/Xlib.h>                                                           
#include <X11/Xatom.h>                                                          
#include <X11/keysym.h>  

Can you recommend any other simple program/hacks for doing
this cool little usefull things in X ? Simple and usefull X11 clipboard hacks ? Or other similar thing for users who prefer simple window managers (I use dwm) instead of heavy desktop managers ?
Where did you find inspiration for this ?

Funny I never heard about it on the Arch Linux forums.
Maybe you could present both your programs there on , 
Community Contributions / Arch Linux Forums
https://bbs.archlinux.org/viewforum.php?id=27

PS: By the way,  apropos felinks

I dont have experience with making AUR packages and compiling.
But I want to test and switch from elinks to felinks on ArchLinux (just like Debian did).
I would love to test this python stuff on felinks, 

> hellricer/soupmonkey: Userscript manager for ELinks powered by BeautifulSoup library.
> https://github.com/hellricer/soupmonkey
> 

I already tried to convince this package to switch to felinks (see my comments there)
https://aur.archlinux.org/packages/elinks-git/

Do you know why the official Arch package, https://www.archlinux.org/packages/community/x86_64/elinks/
still uses elinks instead of felinks (like Debian does) ?

Do you have any interest in felinks for ArchLinux ?

**Comment by sgerwk** [comment]
Hi! Thanks for the suggestions, all very useful indeed.

1. Making a package for aur seems a good idea. Not in a short time because I am not currently using the PC where I run Arch, but I will do that.

2. The README deserves some expansion, you're right. I think I will use your exact suggestion of copying the example from the man page.

3. I remember some programs did nice stuff especially with selections, but I forgot which they are, maybe because I ended up using none of them. I wrote multiselect because of a specific web form I have to fill many times with the very same data over and over again. Some browser extensions do something similar, but I do not believe they are a much general and stable solution.

I never realized that ArchLinux had a forum. Should check that, indeed.

4. At a first glance, I suspect the only way to get felinks on Arch is to create a new package just for felinks.
